settings.gradle

```diff
@@ -1,9 +1,6 @@
 pluginManagement {
     repositories {
         gradlePluginPortal()
```

Comment:
This repository will be applied by default for plugins resolution. So we can also remove this line, as well as whole empty plugin management block.

Reply:
Thanks. You are right. It is as well stated in the docs: https://docs.gradle.org/current/userguide/plugins.html#sec:custom_plugin_repositories